addOns/graphql/src/main/resources/org/zaproxy/zap/extension/graphql/resources/Messages.properties

```diff
@@ -0,0 +1,19 @@
+# This file defines the default (English) variants of all of the internationalised messages
+
+graphql.api.action.importFile = Imports a GraphQL schema from a local file.
+graphql.api.action.importFile.param.file = The file that contains the GraphQL schema.
+graphql.api.action.importFile.param.target = The Target URL to override the server URL present in the schema.
+graphql.api.action.importUrl = Retrieves a GraphQL schema from an endpoint URL (via Introspection)
```

Comment:
Should have a period as well?

Reply:
The cmdline entries below also seem to mix period and not period. (For my 2 cents they should have periods)